SupportClient/SupportClient/Views/Cells/BugsRequestsTableViewCell.swift

```diff
@@ -0,0 +1,77 @@
+//
+//  BugsRequestsTableViewCell.swift
+//  SupportClient
+//
+//  Created by Yana Sang on 5/8/20.
+//  Copyright © 2020 Cornell Appdev. All rights reserved.
+//
+
+import UIKit
+
+class BugsRequestsTableViewCell: UITableViewCell {
+
+    static let reuseIdentifier = "bugsRequestsTableViewCell"
+
+    private let tagsLabel = UILabel()
+    private let timeLabel = UILabel()
+    private let messageLabel = UILabel()
+
+    override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+        super.init(style: style, reuseIdentifier: reuseIdentifier)
+
+        setupViews()
+        setupConstraints()
+    }
+
+    func setupViews() {
```

Comment:
I've personally been separating each UI element into its own function, so I feel like we should standardize it across the codebase, whether with this style or mine.

Reply:
i'll switch over to a function for each UI element